RadixWallet/Features/DisplayEntitiesControlledByMnemonic/DisplayEntitiesControlledByMnemonic+View.swift

```diff
@@ -94,7 +96,11 @@ extension DisplayEntitiesControlledByMnemonic {
 					}
 
 					if viewStore.accounts.isEmpty {
-						NoContentView("Hidden Accounts only")
+						if viewStore.hasHiddenAccounts {
+							NoContentView("Hidden Accounts only.")
+						} else {
+							NoContentView("No accounts yet.")
```

Comment:
will change this a bit. But yeah idea is to not show "Hidden Accounts only" when there are NO accounts at all controlled by this newly generated MAIN BDFS.